docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -210,10 +209,11 @@ The point in time that the graphs start will be your back-filling end time. Chec
 
 Backfilling Process
 -------------------
-backup
+Backup
 ^^^^^^
-If you have a long retention period you are using an external directory that holds the Prometheus data, back it up, in case
-If you have a long retention period, you are using an external directory that holds the Prometheus data back it up; if something goes wrong in the process, you can revert the process.
+Backup any external directory containing Prometheus data; if something goes wrong, you can revert the changes.
+
+The monitoring stack will need to be restarted at least once. This process cannot be completed without an external directory when using the -d command-line option.
```

Comment:
This is confusing, see the original description (droping the duplication).
To complete this process, you need to have the data in an exteranl directory, the way you use an external directory is with the -d command line option.